venues/auai.org/UAI/2017/python/email-lazy-reviewers.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/python
+
+"""
+Sends an email to the members of the group of your choice.
+
+"""
+
+## Import statements
+import argparse
+import csv
+import sys
+import openreview
+import requests
+
+## Handle the arguments
+parser = argparse.ArgumentParser()
+parser.add_argument('-v','--verbose', help="set to true if you want late users listed per-paper")
+parser.add_argument('--baseurl', help="base URL")
+parser.add_argument('--username')
+parser.add_argument('--password')
+
+args = parser.parse_args()
+
+## Initialize the client library with username and password
+client = openreview.Client(baseurl=args.baseurl, username=args.username, password=args.password)
+
+#################################################
+#                                               #
+#   EDIT YOUR MESSAGE AND SUBJECT LINE BELOW    #
+#                                               #
+#################################################
+
+
+subjectline = "<Insert your subject line here>"
+
+message = """
+
+<Insert your multi-line email message here>
+
+"""
+
+
+#################################################
+#                                               #
+#               END OF MESSAGE                  #
+#                                               #
+#################################################
+
+submissions = client.get_notes(invitation='auai.org/UAI/2017/-/blind-submission')
+invitation = 'Submit/Review'
+
+verbose = True if args.verbose and args.verbose.lower()=='true' else False
+
+
+total_missing = 0;
+total_complete = 0;
+
+def get_data(invitation):
+
+    headers = {'User-Agent': 'test-create-script', 'Content-Type': 'application/json', 'Authorization': 'Bearer ' + client.token}
+    anon_reviewers = requests.get(client.baseurl+'/groups?id=auai.org/UAI/2017/Paper.*/AnonReviewer.*', headers = headers)
```

Comment:
I guess you can't use the client.get_groups() method for this?

If that doesn't work, I can add it to my to-do list for openreview-py